### PR TITLE
Adding content to Win forensics and other tools; Adding sections for OSX forensics,Sandboxing/reversing and Books

### PR DESCRIPTION
**Pull request by mikesxrs**
Added misc information I have found useful

-Highlighter from Fire/Mandiant
-RegRipper Registry tool for win forensics
-OSX Evidence Section
-Sandboxing/reversing tools (both local and online)
-Book Section with Amazon link (feel free to change)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,13 +68,29 @@ A curated list of tools and resources for security incident response, aimed to h
...
 
+### OSX Evidence Collection
+* [OSX Auditor](https://github.com/jipegit/OSXAuditor) - OSX Auditor is a free Mac OS X computer forensics tool
+
+### Sandboxing/reversing tools
+* [Cuckoo](https://github.com/cuckoobox) - Open Source Highly configurable sandboxing tool 
+* [Mastiff](https://github.com/KoreLogicSecurity/mastiff) - MASTIFF is a static analysis framework that automates the process of extracting key characteristics from a number of different file formats.
+* [Viper](https://github.com/viper-framework/viper) - Viper is a python based binary analysis and management framework, that works well with Cuckoo and YARA.
+* [Virustotal](Virustotal.com) - Virustotal, a subsidiary of Google, is a free online service that analyzes files and URLs enabling the identification of viruses, worms, trojans and other kinds of malicious content detected by antivirus engines and website scanners
+* [Malwr](malwr.com) - Malwr is a free online malware analysis service and community, which is powered by the Cuckoo Sandbox
```
Comment (meirwah) [inline README.md]: broken link.. you must put full path here :  https://malwr.com/

File: README.md [hunk 2]
```diff
@@ -64,13 +68,29 @@ A curated list of tools and resources for security incident response, aimed to h
...
+* [RegRipper](https://code.google.com/p/regripper/wiki/RegRipper) - Regripper is an open source tool, written in Perl, for extracting/parsing information (keys, values, data) from the Registry and presenting it for analysis.
 
+### OSX Evidence Collection
+* [OSX Auditor](https://github.com/jipegit/OSXAuditor) - OSX Auditor is a free Mac OS X computer forensics tool
+
+### Sandboxing/reversing tools
+* [Cuckoo](https://github.com/cuckoobox) - Open Source Highly configurable sandboxing tool 
+* [Mastiff](https://github.com/KoreLogicSecurity/mastiff) - MASTIFF is a static analysis framework that automates the process of extracting key characteristics from a number of different file formats.
+* [Viper](https://github.com/viper-framework/viper) - Viper is a python based binary analysis and management framework, that works well with Cuckoo and YARA.
+* [Virustotal](Virustotal.com) - Virustotal, a subsidiary of Google, is a free online service that analyzes files and URLs enabling the identification of viruses, worms, trojans and other kinds of malicious content detected by antivirus engines and website scanners
```
Comment (meirwah) [inline README.md]: Broken link here too : need https://virustotal.com

File: README.md [hunk 3]
```diff
@@ -64,13 +68,29 @@ A curated list of tools and resources for security incident response, aimed to h
...
+* [Malwr](malwr.com) - Malwr is a free online malware analysis service and community, which is powered by the Cuckoo Sandbox
 
 ### Other Tools
 * [Hindsight](https://github.com/obsidianforensics/hindsight) - Internet history forensics for Google Chrome/Chromium
 * [Kansa](https://github.com/davehull/Kansa/) - Kansa is a modular incident response framework in Powershell. 
 * [Stalk](https://www.percona.com/doc/percona-toolkit/2.2/pt-stalk.html) - Collect forensic data about MySQL when problems occur.
+* [Etherpad](https://github.com/ether/etherpad-lite) - Good collaboration tool, similar to google doc but doesnt store data in the cloud.
+* [Kibana](https://github.com/elastic/kibana) - Big Data analytics and visualization platform
+* [Elastic Search](https://github.com/elastic/elasticsearch) - Big Data solution for Real-time searching and analytics
```
Comment (meirwah) [inline README.md]: not sure about those tools (Etherpad, Kibana, Elastic) since they are not security focused so much...

**Comment by meirwah** [comment]
Very nice additions!!
For next time try to open a PR for each addition, that way we can merge some of them (in case we are not sure about the others...)

File: README.md (hunk 1 above)
Comment (mikesxrs) [inline README.md]: Thanks for the help

File: README.md (hunk 3 above)
Comment (mikesxrs) [inline README.md]: No worries, didn't know how focused you were looking to make it. Removed for the time being.

**Pull request merged**